core/src/main/java/org/jdbi/v3/core/mapper/reflect/CaseInsensitiveColumnNameMatcher.java

```diff
@@ -23,16 +21,12 @@
 public class CaseInsensitiveColumnNameMatcher implements ColumnNameMatcher {
     @Override
     public boolean columnNameMatches(String columnName, String javaName) {
-        return normalize(columnName).equals(normalize(javaName));
+        return columnName.equalsIgnoreCase(javaName);
     }
 
     @Override
     public boolean columnNameStartsWith(String columnName, String prefix) {
-        return normalize(columnName).startsWith(normalize(prefix));
-    }
-
-    private String normalize(String string) {
-        return string.toLowerCase(Locale.ROOT);
+        return columnName.regionMatches(true, 0, prefix, 0, prefix.length());
```

Comment:
😍

Reply:
I know, I just stumbled across that method for the first time. I've wished for `startWithIgnoreCase()` for a while--turns out we had one already with a silly name